templates/nro/.circleci/config.yml.tmpl

```diff
@@ -479,6 +505,7 @@ workflows:
 
 {{- if isTrue .Env.MAKE_RELEASE }}
   release:
+    unless: << pipeline.parameters.rollback >>
```

Comment:
Maybe there is an alternative to creating the rollback pipeline for a tag? Not sure what happens if we just remove this condition: https://github.com/greenpeace/planet4-nro-generator/pull/33/files?diff=split&w=1#diff-639a8d421e4f81988cc6186eb6894cfbR559 . Then we wouldn't have to add this check both here and in the inverse in the rollback workflow.

Reply:
I don't think there is an alternative way, besides using the filters. If we remove that, then it will run every time for each branch.